### ISSUE TITLE
Some dimension error in cpea.py

### ISSUE THREAD

**Issue by lxy-146** [FushengHao/CPEA#6]
In line 57, the dimension of feat_shot is S x K x n x C instead of K x S x n x C
Meanwhile , in line 58, it should be K x n x C
If my understanding is incorrect, please correct me. 
Thx.

**Comment by FushengHao** [comment]
Thanks! The dimension of feat_shot is related to the acquisition of few-shot tasks.

For example, the labels of a 5way-2shot task are organized as follows: labels = torch.tensor([0, 1, 2, 3, 4, 0, 1, 2, 3, 4]), and the average operation along the shot dimension for labels can be processed as follows: labels.reshape(2,-1).mean(dim=0) # labels.reshape(shot,-1).mean(dim=0).

Similarly, the average operation along the shot dimension for embeddings can be understood in the same way.

**Comment by lxy-146** [comment]
Thank you for your timely response. 

There is no issue with the dimensions of the code itself, but there might be a misunderstanding in the comments on line 57 and 58. Considering a K-way S-shot few-shot task, the average should be taken over the S dimension for `feat_shot` rather than the K dimension, and the code does implement it this way. However, the comment suggests that the result after averaging is S x n x C.

**Comment by FushengHao** [comment]
Thanks! In my opinion, there is a misunderstanding about the meaning of "S".

Our understanding of the meaning of "S" is as follows: "S" means "support" (rather than "shot"), which is used to indicate "way". Correspondingly, "K" is used to indicate "shot". In other words, a few-shot task is S-way K-shot in our case (rather than K-way S-shot).
